lib/curl_setup.h

```diff
@@ -809,4 +809,12 @@ int getpwuid_r(uid_t uid, struct passwd *pwd, char *buf,
 #define ENABLE_QUIC
 #endif
 
+#if defined(USE_UNIX_SOCKETS)
+#if defined(WINAPI_FAMILY) && (WINAPI_FAMILY == WINAPI_FAMILY_APP)
+#  include <afunix.h>
```

Comment:
line 813~814 are moved from `lib/curl_addrinfo.c`. But I think `unixsockets_win32.h` can also fix uwp build issue. Should I remove the UWP-sepcific codes?

Reply:
Do what you think is best there. It seems we don't have anyone around who knows/cares about UWP to help answer this!